tests/regression_tests/mgxs_library_ce_to_mg_nuclides/test.py

```diff
@@ -0,0 +1,85 @@
+import os
+
+import openmc
+import openmc.mgxs
+from openmc.examples import pwr_pin_cell
+
+from tests.testing_harness import PyAPITestHarness
+from tests.regression_tests import config
+
+
+class MGXSTestHarness(PyAPITestHarness):
```

Comment:
This file is almost exactly the same as `mgxs_library_ce_to_mg/test.py`. I think we could potentially combine these, but it would take a bit of refactoring so I won't ask you to do that as part of this PR. Let's just create an issue afterwards to keep track of this.

Reply:
I had the same thought. I'll open an issue now.